com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.PostProcess.cs

```diff
@@ -286,33 +284,23 @@ void BeginPostProcessFrame(CommandBuffer cmd, HDCamera camera, HDRenderPipeline
             CheckRenderTexturesValidity();
 
             // Handle fixed exposure & disabled pre-exposure by forcing an exposure multiplier of 1
-            if (!m_ExposureControlFS)
-            {
-                cmd.SetGlobalTexture(HDShaderIDs._ExposureTexture, m_EmptyExposureTexture);
-                cmd.SetGlobalTexture(HDShaderIDs._PrevExposureTexture, m_EmptyExposureTexture);
-            }
-            else
             {
                 // Fix exposure is store in Exposure Textures at the beginning of the frame as there is no need for color buffer
                 // Dynamic exposure (Auto, curve) is store in Exposure Textures at the end of the frame (as it rely on color buffer)
                 // Texture current and previous are swapped at the beginning of the frame.
                 bool isFixedExposure = IsExposureFixed(camera);
-                if (isFixedExposure)
+
+                //if exposure comes from the parent camera, it means we dont have to calculate / force it.
+                //Its already been done in the parent camera.
+                if (isFixedExposure && camera.exposureControlFS && camera.currentExposureTextures.useCurrentCamera && camera.currentExposureTextures.current != null)
```

Comment:
I'd like these conditions a bit more condensed (maybe in a separete fun call like the IsExposureFixed one)